spec/models/spree/calculator/flat_percent_item_total_spec.rb

```diff
@@ -4,13 +4,20 @@
   let(:calculator) { Spree::Calculator::FlatPercentItemTotal.new }
   let(:line_item) { instance_double(Spree::LineItem, amount: 10) }
 
-  before { calculator.stub :preferred_flat_percent => 10 }
+  before { calculator.stub preferred_flat_percent: 10 }
 
-  it "should compute amount correctly for a single line item" do
+  it "computes amount correctly for a single line item" do
```

Comment:
thanks! I prefer examples this way. `should` doesn't inspire confidence :trollface: